src/main/java/com/applicaster/playerevents/PlayerEventsManager.kt

```diff
@@ -16,8 +16,8 @@ object PlayerEventsManager {
         }
     }
 
-    fun onPlayerEvent(event: String, properties: Any) {
-        playerEventsProviders.forEach { it.onPlayerEvent(event,properties) }
+    fun onPlayerEvent(event: String, properties: HashMap<String,Any>) {
+        playerEventsProviders.forEach { it.onPlayerEvents(event,properties) }
```

Comment:
Can you name it onPlayerEvents in singular, without the s, as we are receiving just one event.

Reply:
will be fixed